src/pytorch_lightning/strategies/horovod.py

```diff
@@ -111,8 +113,10 @@ def _unpack_lightning_optimizer(opt):
         lr_scheduler_configs = self.lr_scheduler_configs
         for config in lr_scheduler_configs:
             scheduler = config.scheduler
+            assert isinstance(scheduler, _LRScheduler), "ReduceLROnPlateau does not have attribute 'base_lr'"
```

Comment:
The assertion does not relate to the attribute base_lr. 
The class of the scheduler is also not ReduceLROnPlateau in general. 
I suggest you remove the comment

Reply:
Sorry, indeed I made a typo, it is ```base_lrs```.
According to the type of ```scheduler``` being ```Union[_LRScheduler, ReduceLROnPlateau]``` If the type is not ```_LRScheduler``` then it is ```ReduceLROnPlateau```, right? 
Do you mean that it is also used with other schedulers not listed is that ```Union```?
In that case shouldn't we update the ```scheduler``` type in the data class ```LRSchedulerConfig```?